openapi.yml

```
@@ -543,6 +543,42 @@ paths:
       tags:
         - End User
       parameters: []
+  "/api/v1/admin/users/{userId}":
+    parameters:
+      - schema:
+          type: string
+        name: userId
+        in: path
+        required: true
+    post:
+      summary: Update User
+      operationId: post-api-v1-admin-users-userId
```

nitpick: we should give these operations shorter names, so that the generated React Hooks and Go endpoints are more concise.

```suggestion
      operationId: update-user
```